### ISSUE TITLE
The curl example here is wrong

### ISSUE THREAD

**Issue by a1300** [GNYIO/gny-docs#63]
Reported by Telegram @MorinelloA 

```
https://docs.gny.io/http/api.html#_2-4-10-check-delgate-forging-status
I'm guessing it should be curl -X GET 'http://localhost:4096/api/delegates/forging/status?publicKey=ff47c9e9bafcf28ae8528c2b259661ade96a3030ab73ddde82b52ee44c9122b5'
```

**Comment by a1300** [comment]
Thanks @MorinelloA

**Comment by a1300** [comment]
Closed by PR: #71
